presto-docs/src/main/sphinx/connector/druid.rst

```diff
@@ -0,0 +1,26 @@
+===============
+Druid Connector
+===============
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+The Druid Connector allows querying `Apache Druid <https://druid.apache.org/>`_ data from Presto.
```

Comment:
Should be above contents list and then drop Overview section title..

Reply:
Done